src/dictionary.rs

```diff
@@ -184,10 +204,136 @@ impl<T: 'static> DictionaryEntry<T> {
         let pfp: *mut [Word; 0] = addr_of_mut!((*ptr).parameter_field);
         NonNull::new_unchecked(pfp.cast::<Word>())
     }
+
+    pub fn parameters(&self) -> &[Word] {
```

Comment:
```suggestion
    /// Obtain the array of parameters for a given entry
    ///
    /// Similar to `DictionaryEntry::pfa()`, but more convenient to use
    /// when the parameters have already been initialized 
    pub fn parameters(&self) -> &[Word] {
```

Reply:
@hawkw I think pfa exists for reasons of initialization (it uses addr of trickery). Honestly I should probably un-`pub` it, and maybe not mention `pfa` at all.